src/main/java/com/gregtechceu/gtceu/common/machine/electric/WorldAcceleratorMachine.java

```diff
@@ -82,19 +87,16 @@ public class WorldAcceleratorMachine extends TieredEnergyMachine implements ICon
     private boolean active = false;
     private TickableSubscription tickSubs;
 
-    public WorldAcceleratorMachine(IMachineBlockEntity holder, int tier, Object... args) {
-        super(holder, tier, GTMachineUtils.defaultTankSizeFunction, args);
+    public WorldAcceleratorMachine(BlockEntityCreationInfo info, int tier) {
+        super(info, tier, (TieredEnergyMachine machine) -> {
+            long tierVoltage = GTValues.V[machine.getTier()];
+            return new NotifiableEnergyContainer(machine, tierVoltage * 256L, tierVoltage, 8, 0L, 0L);
+        });
```

Comment:
These are getting worse and worse. If calling `super()` requires passing in a lambda with a multi-line body, something is definitely wrong. At this point, maybe machines need a factory method and these various containers/handlers passed via dependency injection into the constructor instead.